backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/export/SVGExportContext.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.spring.collaborative.diagrams.export;
+
+import java.util.Collection;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import org.eclipse.sirius.web.diagrams.Diagram;
+import org.eclipse.sirius.web.diagrams.Node;
+import org.eclipse.sirius.web.spring.collaborative.diagrams.export.api.IExportContext;
+import org.eclipse.sirius.web.spring.collaborative.diagrams.export.api.IExportHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+/**
+ * Component used to export the elements of the model to their SVG format.
+ *
+ * @author rpage
+ */
+@Service
+public class SVGExportContext implements IExportContext {
```

Comment:
I'm not sure why this concept is necessary? Why can't the export service call the DiagramExportHandler which would in turn call the NodeExportHandler and EdgeExportHandler? Why do you need that context concept with this generic approach while we know exactly the shape of a diagram?